jack-core/src/com/rapleaf/jack/transaction/TransactorImpl.java

```diff
@@ -11,20 +11,28 @@
 import com.rapleaf.jack.exception.SqlExecutionFailureException;
 
 /**
- * If there is any exception while executing the query, throws {@link com.rapleaf.jack.exception.SqlExecutionFailureException}.
+ * If there is any exception while executing the query, throws
+ * {@link com.rapleaf.jack.exception.SqlExecutionFailureException}.
  * <p>
  * If there is no available connections, throws {@link com.rapleaf.jack.exception.NoAvailableConnectionException}.
  * Users can either increase the max total connections or max wait time.
  * <p>
  * If the DB manager has already been closed, throws {@link IllegalStateException}.
  * <p>
- * If new DB connections cannot be created, throws {@link com.rapleaf.jack.exception.ConnectionCreationFailureException}.
+ * If new DB connections cannot be created, throws
+ * {@link com.rapleaf.jack.exception.ConnectionCreationFailureException}.
  */
 public class TransactorImpl<DB extends IDb> implements ITransactor<DB> {
   private static final Logger LOG = LoggerFactory.getLogger(TransactorImpl.class);
 
   private final IDbManager<DB> dbManager;
 
+  private TransactorMetrics queryMetrics = new TransactorMetricsImpl(10);
+
+  /*
+  TODO Do I need to initialize it in the builder??
+   */
```

Comment:
We put a parameter in the builder if we allow the user to customize that parameter. In this case, for now we don't want users to manipulate this object. So it's fine to construct it here.